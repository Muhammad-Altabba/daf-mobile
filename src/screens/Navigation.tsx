import React from 'react'
import {
  createDrawerNavigator,
  createAppContainer,
  createStackNavigator,
} from 'react-navigation'
import Settings from './Settings'
import Welcome from './Welcome'
import Logs from './Logs'
import Analytics from 'appcenter-analytics'

Analytics.setEnabled(true)

const LogsNavigator = createStackNavigator({
  LogsScreen: {
    screen: Logs,
    navigationOptions: {
      title: 'Logs',
    },
  },
})

const TabNavigator = createDrawerNavigator({
  Home: Welcome,
  Logs: LogsNavigator,
  Settings: Settings,
})

export default createAppContainer(TabNavigator)
