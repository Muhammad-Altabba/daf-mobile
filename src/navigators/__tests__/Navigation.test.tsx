import 'react-native'
import React from 'react'
import Navigation from '../'
import { render } from 'react-native-testing-library'
jest.useFakeTimers()

describe('Navigation', () => {
  it('renders correctly', () => {
    const tree = render(<Navigation />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
