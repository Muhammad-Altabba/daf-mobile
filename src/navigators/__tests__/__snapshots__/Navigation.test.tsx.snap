// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation renders correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    collapsable={false}
    onGestureHandlerEvent={[Function]}
    onGestureHandlerStateChange={[Function]}
    style={
      Object {
        "backgroundColor": "#eee",
        "flex": 1,
        "flexDirection": "column-reverse",
        "overflow": "hidden",
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        pointerEvents="auto"
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "opacity": 1,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "transform": Array [
              Object {
                "translateX": 0,
              },
              Object {
                "translateY": 0,
              },
            ],
          }
        }
      >
        <View
          accessibilityElementsHidden={false}
          style={
            Object {
              "backgroundColor": "#fff",
              "flex": 1,
            }
          }
        >
          <View
            pointerEvents="auto"
            style={
              Object {
                "alignItems": undefined,
                "backgroundColor": undefined,
                "borderRadius": undefined,
                "flex": 1,
                "flexDirection": undefined,
                "height": undefined,
                "justifyContent": undefined,
                "margin": undefined,
                "marginBottom": undefined,
                "marginLeft": undefined,
                "marginRight": undefined,
                "marginTop": undefined,
                "padding": undefined,
                "paddingBottom": undefined,
                "paddingHorizontal": undefined,
                "paddingLeft": undefined,
                "paddingRight": undefined,
                "paddingTop": undefined,
                "width": undefined,
              }
            }
          >
            <View
              pointerEvents="auto"
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": undefined,
                  "borderRadius": undefined,
                  "flex": 1,
                  "flexDirection": undefined,
                  "height": undefined,
                  "justifyContent": "center",
                  "margin": undefined,
                  "marginBottom": undefined,
                  "marginLeft": undefined,
                  "marginRight": undefined,
                  "marginTop": undefined,
                  "padding": undefined,
                  "paddingBottom": undefined,
                  "paddingHorizontal": undefined,
                  "paddingLeft": undefined,
                  "paddingRight": undefined,
                  "paddingTop": undefined,
                  "width": undefined,
                }
              }
            >
              <ActivityIndicator
                animating={true}
                color="#3A89C9"
                hidesWhenStopped={true}
                size="large"
              />
            </View>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
